lib/factory-helper/team.rb

```diff
@@ -12,7 +12,7 @@ def creature
       end
 
       def state
-        fetch('factory_helper.address.state').titleize
+        fetch('address.state').split(' ').map(&:capitalize).join(' ')
```

Comment:
This ain't Rails :smile: The Team subclass overall is not well implemented and is mostly untested. Added task to Trello.